src/EnvelopeCreator/SendEnvelope.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DocusignBundle\EnvelopeCreator;
+
+use DocuSign\eSign\Api\EnvelopesApi;
+use DocuSign\eSign\ApiClient;
+use DocuSign\eSign\Configuration;
+use DocusignBundle\EnvelopeBuilder;
+use DocusignBundle\Grant\GrantInterface;
+
+class SendEnvelope
+{
+    public $grant;
+
+    public function __construct(GrantInterface $grant)
+    {
+        $this->grant = $grant;
+    }
+
+    /**
+     * @throws \DocuSign\eSign\ApiException
+     */
+    public function handle(EnvelopeBuilder $envelopeBuilder): void
```

Comment:
```suggestion
    public function __invoke(EnvelopeBuilder $envelopeBuilder): void
```

As all your services match `__invoke(EnvelopeBuilder $envelopeBuilder)`, they could all implement the same interface, like `EnvelopeBuilderCallableInterface`.

But as the EnvelopeBuilder is created per document (cf. config), all those services must be declared the same (cf. DocusignExtension)